src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -182,7 +182,7 @@ foam.CLASS({
           window.history.replaceState(null, null, window.location.origin);
           location.reload();
         } else {
-          await this.auth.login(x, this.email, this.desiredPassword);
+          await this.auth.login(x, this.userName, this.desiredPassword);
```

Comment:
not very familiar with this code. How do we know whether we want to send email or username?

Reply:
username is required and must be unique.